example/ios/ClevertapSignedCallReactNativeExample/Info.plist

```diff
@@ -17,11 +17,15 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.0.1</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>4</string>
```

Comment:
@Sonal-Kachare Do not add credentials in the final commit

Reply:
this is done.